apps/coscrad-frontend/src/utils/generic-components/presenters/external-link-presenter.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Link, Tooltip, Typography } from '@mui/material';
+
+interface ExternalLinkProps {
+    url: string;
+}
+
+export const ExternalLinkPresenter = ({ url }: ExternalLinkProps): JSX.Element => {
+    const linkDisplayLength = 30;
+
+    const displayURL =
+        url.length > linkDisplayLength ? `${url.substring(0, linkDisplayLength)}...` : url;
```

Comment:
You  might want to break out a utility
```ts
condenseText = (text: string, maxNumberOfChars: number, placeholderText: string = '...') => {...}
```
It sounds like we have other use cases, and it really helps with the readability. I find this line too low-level.